### DISCUSSION TITLE
How To Change Custom Color Created In SASS For Dark Mode

### DISCUSSION THREAD

**Discussion by clarnx** [twbs/bootstrap#37838] [Q&A]
Hello I am using  bootstrap@5.3.0-alpha1 and I have created a custom color in sass. Now when I switch to dark mode the color is the same.
How do I change the custom color for dark mode.
Kindly check the code below.

```
@import "./bootstrap_default.scss";
@import "bootstrap/scss/functions";
@import "bootstrap/scss/variables";

$custom-colors: (
    "my-custom-color": #17c964,
);

// Rebuild all the bg-* and text-* classes
$theme-colors: map-merge($theme-colors, $custom-colors);
$theme-colors-rgb: map-loop($theme-colors, to-rgb, "$value");

@import "bootstrap/scss/maps";
@import "bootstrap/scss/mixins";
@import "bootstrap/scss/utilities";
@import "bootstrap/scss/utilities/api";
@import "bootstrap/scss/bootstrap";
```

**Comment by julien-deramond** [comment]
Hello @clarnx
I've tried the following PR to tackle this topic the other day: https://github.com/twbs/bootstrap/pull/37737.
It's heavily WIP but hope it can help.

**Reply by clarnx** [reply]
Hi @julien-deramond thanks for the suggestion. 
I read suggestions in the issue #37737 however I am having a hard time understanding it as my sass knowledge is basic.

If possible can you show a code sample of how to change the dark mode version of a custom color created called "my-custom-color" using the code in #37737 

Thanks.

**Reply by julien-deramond** [reply]
If I'm not mistaken there's no switch for background colors in dark mode but only for background subtle colors.
I've done it quickly but if you want a background subtle custom color in light/dark mode, you can do something like this: https://stackblitz.com/edit/github-csdted?file=src/scss/styles.scss.
Color mode (dark mode) is still in alpha so it's not yet finalized.
/cc @mdo: maybe there's a simpler way of doing it, but at some point IMO it'll need to be simplified
